.github/workflows/ci-aarch64-nightly.yml

```diff
@@ -0,0 +1,132 @@
+# *******************************************************************************
+# Copyright 2024 Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+
+name: "CI AArch64 (Nightly)"
```

Comment:
(nit) Suggest 'Nightly AArch64' for workflow name and 'nightly-aarch64.yml' for the file name.

Reply:
Done. Any more changes and I'll have to change the copyright header to 2025..